docs/guides/_toc.json

```diff
@@ -505,7 +505,17 @@
         },
         {
           "title": "Qiskit addons",
-          "url": "/guides/addons"
+          "children": [
+            {
+              "title": "Qiskit addons overview",
+              "url": "/guides/addons"
+            },
+            {
+              "title": "Sample-based Quantum Diagonalization",
+              "url": "/guides/sqd-intro"
```

Comment:
Should be qiskit-addons-sqd. Then additional pages will be qiskit-addons-sqd-my-topic.